scripts/mgear/maya/shifter/gameTools.py

```diff
@@ -187,61 +183,65 @@ def exportAssetAssembly(name, rigTopNode, meshTopNode, path, postScript=None):
     if pm.ls(rigTopNode):
         rigTopNode = pm.PyNode(rigTopNode)
     else:
-        pm.displayError("{} doesn't exist or duplicated. Please check your scene".format(rigTopNode))
+        pm.displayError(
+            "{} doesn't exist or duplicated. Please check your "
+            "scene".format(rigTopNode))
         return
 
     if pm.ls(meshTopNode):
         meshTopNode = pm.PyNode(meshTopNode)
     else:
-        pm.displayError("{} doesn't exist or duplicated. Please check your scene".format(meshTopNode))
+        pm.displayError(
+            "{} doesn't exist or duplicated. Please check "
+            "your scene".format(meshTopNode))
         return
     # check the folder and script
     # if the target name exist abort and request another name
 
     deformer_jnts = rigTopNode.rigGroups[3].connections()[0].members()
     if not deformer_jnts:
-        pm.displayError("{} is empty. The tool can't find any joint".format(meshTopNode))
-
+        pm.displayError(
+            "{} is empty. The tool can't find any joint".format(meshTopNode))
 
     # export connections and cut joint connections
 
     # cut al possible remaining connection and adjust hierarchy
 
-
-
-    #pot script
+    # pot script
```

Comment:
Pot or Post? :)

Reply:
should be post, this a typo. 
However this tool is not finished. I have put this on hold until finish this refactor